plugin/client_rpc.go

```diff
@@ -525,8 +525,8 @@ func (s *hooksRPCServer) FileWillBeUploaded(args *Z_FileWillBeUploadedArgs, retu
 	return nil
 }
 
-// MessageWillBePosted is in this file because of the difficulty of identifiying which fields need special behaviour.
-// The special behaviour needed is decoding the returned post into the original one to avoid the unintentional removal
+// MessageWillBePosted is in this file because of the difficulty of identifying which fields need special behavior.
```

Comment:
:+1: identifiying/identifying, but behaviour is a valid spelling in BE

Reply:
Sorry, should I change "behavior" back to "behaviour" ??

Reply:
Yes please

Reply:
There are still two instances of `behavior`